Cargo.toml

```diff
@@ -12,17 +12,20 @@ dag-jose = "0.1"
 futures = "0.3"
 futures-util = "0.3"
 hex = "0.4.3"
-iroh-metrics = { git = "https://github.com/3box/beetle", branch = "main" }
-iroh-p2p = { git = "https://github.com/3box/beetle", branch = "main" }
-iroh-rpc-client = { git = "https://github.com/3box/beetle", branch = "main" }
-iroh-rpc-types = { git = "https://github.com/3box/beetle", branch = "main" }
-iroh-store = { git = "https://github.com/3box/beetle", branch = "main" }
-libipld = "0.15"                                                               # use same version as Iroh
-libp2p = { version = "0.50", default-features = false }                        # use same version as Iroh
-libp2p-tls = { version = "=0.1.0-alpha", default-features = false }            # use explicit version of dep to avoid conflict
-multiaddr = "0.16"                                                             # use same version as Iroh
+iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
```

Comment:
Yeah we won't merge with this line still inplace (see PR TODO). We will first merge the `custom-behaviour` branch and then this can point back to the `main` branch.